src/imports.rs

```diff
@@ -111,7 +111,13 @@ fn compare_use_items(context: &RewriteContext, a: &ast::Item, b: &ast::Item) ->
             Some(compare_use_trees(a_tree, b_tree, false))
         }
         (&ast::ItemKind::ExternCrate(..), &ast::ItemKind::ExternCrate(..)) => {
-            Some(context.snippet(a.span).cmp(context.snippet(b.span)))
+            let get_name = |sp: Span| {
+                // The span for extern includes the trailing comma.
```

Comment:
IS this a typo? Should be semicolon, rather than comma.

Also, what if the user wrote `extern crate foo ;` or `foo         ;` would this still work?